Challenge #29 (Real-life Application): Application doesn't serve

When running `npx nx serve testing-real-life-application` the application doesn't serve.

> Error: apps/testing/29-real-life-application/src/app/list/ticket.store.ts:6:3 - error TS2305: Module '"@ngrx/component-store"' has no exported member 'tapResponse'.

The incorrect imports, such as `concatLatestFrom` and `tapResponse`, should point to `@ngrx/operators`.

--- a/apps/testing/29-real-life-application/src/app/detail/detail.store.ts
+++ b/apps/testing/29-real-life-application/src/app/detail/detail.store.ts
@@ -1,11 +1,7 @@
 import { Injectable } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import {
-  ComponentStore,
-  OnStateInit,
-  tapResponse,
-} from '@ngrx/component-store';
-import { concatLatestFrom } from '@ngrx/effects';
+import { ComponentStore, OnStateInit } from '@ngrx/component-store';
+import { concatLatestFrom, tapResponse } from '@ngrx/operators';
 import { pipe } from 'rxjs';
 import { map, mergeMap, tap } from 'rxjs/operators';
 import { PARAM_TICKET_ID } from '../app.route';

--- a/apps/testing/29-real-life-application/src/app/list/ticket.store.ts
+++ b/apps/testing/29-real-life-application/src/app/list/ticket.store.ts
@@ -3,8 +3,8 @@ import {
   ComponentStore,
   OnStateInit,
   OnStoreInit,
-  tapResponse,
 } from '@ngrx/component-store';
+import { tapResponse } from '@ngrx/operators';
 import { pipe } from 'rxjs';
 import { mergeMap, tap } from 'rxjs/operators';
 import { BackendService, Ticket, User } from '../backend.service';

